editoast/src/railjson/mod.rs

```diff
@@ -49,6 +50,54 @@ pub struct TrackSection {
     pub sch: LineString,
 }
 
+#[derive(Debug, Derivative, Clone, Deserialize, Serialize)]
+#[derivative(Default)]
+pub struct Signal {
+    #[derivative(Default(value = "\"my_signal\".to_string()"))]
+    pub id: String,
+    #[derivative(Default(value = "ObjectRef {
+        obj_type: ObjectType::TrackSection,
+        obj_id: \"my_track\".to_string(),
+    }"))]
```

Comment:
You can oneline this and use `r#"..."#` to avoid escape (https://doc.rust-lang.org/reference/tokens.html#raw-string-literals)